static/css/report.css

```diff
@@ -56,24 +51,16 @@ body {
 	width: 100%;
 }
 
-table.centeraligned td{
-	text-align:center;
-}
-table.centeraligned th{
+.centeraligned {
 	text-align:center;
 }
+
 #dailystats-placeholder td.tdborder {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (td.tdborder) is overqualified, just use .tdborder without element name.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/946152948/issues/source?bid=2452379&fileBranchId=2849128#l58)